### PR TITLE
Add information which file was not found

### PR DESCRIPTION
**Pull request by pustovalov**
before:
```
  library/dist/index.js
  Size limit: 85 KB
  Size:       79.83 KB with all dependencies, minified and gzipped

  Size limit: 85 KB
  Size:       0 B   with all dependencies, minified and gzipped

  Size limit: 100 KB
  Size:       0 B    with all dependencies, minified and gzipped
```

after:
```
  library/dist/index.js
  Size limit: 85 KB
  Size:       79.83 KB with all dependencies, minified and gzipped

  File not found: library/dist/index2.js
  Size limit: 85 KB
  Size:       0 B   with all dependencies, minified and gzipped

  File not found: public/packs-test/lite-*.js
  Size limit: 100 KB
  Size:       0 B    with all dependencies, minified and gzipped
```

### CODE REVIEW

File: packages/size-limit/get-config.js [hunk 1]
```diff
@@ -109,6 +109,7 @@ module.exports = async function getConfig (plugins, process, args, pkg) {
     config.checks = await Promise.all(
       result.config.map(async check => {
         if (check.path) {
+          check.filePath = check.path
           check.path = await globby(check.path, { cwd: config.cwd })
```
Comment (ai) [inline packages/size-limit/get-config.js:113]: Can I ask you to refactor it a little. It was a mistake to use `path` for pattern and for the files.

Can you replace it to:

```js
          check.files = await globby(check.path, { cwd: config.cwd })
```

Comment (pustovalov) [reply packages/size-limit/get-config.js:113]: I will try, but it will affect not only one place)

**Comment by pustovalov** [comment]
strange, but locally tests always pass

**Comment by pustovalov** [comment]
made some checks:

`master`
4 of 11 failed 
https://monosnap.com/file/GUzSg4IPLACH5tLbXUPyPjTlYN5q3Y

https://travis-ci.com/github/pustovalov/size-limit/builds/179049717
https://travis-ci.com/github/pustovalov/size-limit/builds/179049737
https://travis-ci.com/github/pustovalov/size-limit/builds/179049758
https://travis-ci.com/github/pustovalov/size-limit/builds/179049917

`file-not-found`
4 of 11 failed
https://monosnap.com/file/ndZWwmR0fhGrqW3cewxS4pG2pFV3EG

https://travis-ci.com/github/pustovalov/size-limit/builds/179050156
https://travis-ci.com/github/pustovalov/size-limit/builds/179050224
https://travis-ci.com/github/pustovalov/size-limit/builds/179050232
https://travis-ci.com/github/pustovalov/size-limit/builds/179050258


`add jest.useFakeTimers`
1 of 11 failed
https://monosnap.com/file/lDCufiQBwftRV9kLS6y9tHLispq4nH

https://travis-ci.com/github/pustovalov/size-limit/builds/179051297 - different error, trouble with estimo

**Comment by ai** [comment]
Wow! Thanks.

Seems like I just forget to merge PR. I will release it today.

**Comment by ai** [comment]
Released in 4.5.6

**Pull request merged**